### PR TITLE
[FR] Update "Skinning history" and "osu! skinning"

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Skinning/osu!/fr.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ Notes :
   - Pour que cet underlay soit le numéro du combo, mettez `HitCircleOverlayAboveNumber` à `0`.
 - Également utilisé pour `sliderstartcircleoverlay` et/ou `sliderendcircleoverlay` si non skinné.
 - Cela devrait être un cercle.
+- Cet élément animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Pisapou) [inline wiki/Skinning/osu!/fr.md:256]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```

File: wiki/Skinning/osu!/fr.md [hunk 2]
```diff
@@ -341,6 +342,7 @@ Notes :
 - Remplace l'image `hitcircle.png` pour le début du slider.
 - `sliderstartcircle.png` est nécessaire pour que cela fonctionne.
 - Cela devrait être un cercle.
+- Cet élément animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Pisapou) [inline wiki/Skinning/osu!/fr.md:345]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```

File: wiki/Skinning/osu!/fr.md [hunk 3]
```diff
@@ -364,20 +366,18 @@ Notes :
...
-- Taux d'animation : 2 FPS (4 FPS max).
-  - Ce taux est affecté par les mods Half Time et Double Time/Nightcore.
 - Cet élément s'estompe avant d'être terminé et s'étend une fois terminé.
   - Si le mode [Hidden](/wiki/Game_modifier/Hidden) est activé, il s'affichera en fondu avant de se terminer et ne disparaîtra qu'après.
 - Le numéro de la combinaison peut être superposé ou non. Par défaut, il est toujours superposé.
   - Pour que cette superposition soit le numéro du combo, mettez `HitCircleOverlayAboveNumber` à `0`.
 - Remplace l'image `hitcircle.png` pour le début du slider.
 - `sliderendcircle.png` est nécessaire pour que cela fonctionne.
 - Cela devrait être un cercle.
+- Cet élément animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Pisapou) [inline wiki/Skinning/osu!/fr.md:380]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```

File: wiki/Skinning/osu!/fr.md [hunk 4]
```diff
@@ -341,6 +342,7 @@ Notes :
 - Remplace l'image `hitcircle.png` pour le début du slider.
 - `sliderstartcircle.png` est nécessaire pour que cela fonctionne.
 - Cela devrait être un cercle.
+- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:345]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History).
```

File: wiki/Skinning/osu!/fr.md [hunk 5]
```diff
@@ -364,20 +366,18 @@ Notes :
...
-- Taux d'animation : 2 FPS (4 FPS max).
-  - Ce taux est affecté par les mods Half Time et Double Time/Nightcore.
 - Cet élément s'estompe avant d'être terminé et s'étend une fois terminé.
   - Si le mode [Hidden](/wiki/Game_modifier/Hidden) est activé, il s'affichera en fondu avant de se terminer et ne disparaîtra qu'après.
 - Le numéro de la combinaison peut être superposé ou non. Par défaut, il est toujours superposé.
   - Pour que cette superposition soit le numéro du combo, mettez `HitCircleOverlayAboveNumber` à `0`.
 - Remplace l'image `hitcircle.png` pour le début du slider.
 - `sliderendcircle.png` est nécessaire pour que cela fonctionne.
 - Cela devrait être un cercle.
+- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:380]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History).
```

File: wiki/Skinning/osu!/fr.md [hunk 6]
```diff
@@ -253,6 +253,7 @@ Notes :
   - Pour que cet underlay soit le numéro du combo, mettez `HitCircleOverlayAboveNumber` à `0`.
 - Également utilisé pour `sliderstartcircleoverlay` et/ou `sliderendcircleoverlay` si non skinné.
 - Cela devrait être un cercle.
+- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History)
```
Comment (Ganondorf-1) [inline wiki/Skinning/osu!/fr.md:256]:
```suggestion
- Cet élément était animable dans le passé. Pour plus de détails, voir [l'histoire du skinning](/wiki/Skinning/History).
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Comment by cl8n** [comment]
I kept `outdated_translation` but removed `outdated_since` from osu skinning cuz it doesn't seem to match anything useful. there's at least a few more parts that don't match en still

**Review by cl8n — APPROVED** [review]

**Pull request merged**